components/wpt-results.html

```diff
@@ -0,0 +1,107 @@
+<!--
+Copyright 2017 Google Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+<link rel="import" href="/bower_components/polymer/polymer-element.html">
+<link rel="import" href="/bower_components/polymer/lib/elements/dom-repeat.html">
+
+<dom-module id="wpt-results">
+  <template>
+    <style>
+      :host {
+        display: block;
+      }
+      .WPTResults__spec {
+        background-color: #eee;
+        padding: 1em;
+        margin-bottom: 1em;
+        border: solid 1px #ccc;
+      }
+    </style>
+
+    <template is="dom-repeat" items="{{specs}}">
+      <div class="WPTResults__spec">
+        <b>{{item.spec}}</b><br/>
+        {{item.passing}} / {{item.total}}<br/>
+        {{_passingPercent(item)}}%
+      </div>
+    </template>
+  </template>
+
+  <script>
+    class WPTResults extends Polymer.Element {
+      static get is() { return 'wpt-results' }
+
+      static get properties() {
+        return {
+          specs: {
+            type: Array,
+            observer: '_specsChanged',
+            value: []
+          }
+        };
+      }
+
+      connectedCallback() {
+        super.connectedCallback();
+
+        this._fetchResults()
+      }
+
+      _specFromTestPath(path) {
+        return path.split('/')[1]
+      }
+
+      async _fetchResults() {
+        const url = 'https://storage.googleapis.com/wptdashboard.appspot.com/results/e2d2cde03d/chrome-60.0-debian-8.json.gz'
+        const response = await fetch(url)
+        const data = await response.json()
```

Comment:
I'm new to async/await and some other stuff, so I have some questions:

Do we want to use "await"  for fetch() and response.json() in order to make this not wait and block the main JS thread? (Is there a "main JS thread")?
Do we declare this function "async" in order to use "await" inside of it?

At some point after fetching the results, the value of this.specs changes, and then this._specsChanged is called. Are the HTML elements actually displaying the results in the template above also updated then? If so, is that independent of _specsChanged?